src/tests.c

```diff
@@ -47,6 +47,37 @@ static void uncounting_illegal_callback_fn(const char* str, void* data) {
     (*p)--;
 }
 
+void max_fe_for_magnitude(secp256k1_fe *r, int m) {
```

Comment:
I think it would be cleaner to have this function be in the respective field_* files, and documented in field.h (it can still be surrounded by `#ifdef VERIFY` there).

Reply:
Is tests.c only ever compiled with VERIFY defined?

Reply:
It is right now, though in #1037 I suggested that maybe we also want tests that aren't. We can deal with that later, though.

Reply:
I was asking myself the same question a few minutes ago. I think the answer is yes but we use  `#ifdef VERIFY` in a few other places (also introduced by you ;)), so it may be good to keep it like this. In fact I think we should run the tests also without `VERIFY`, e.g., to catch compiler bugs that don't appear when `VERIFY` is defined.

Reply:
I've replaced the test method with a new field method _fe_get_bounds when VERIFY is defined. If the tests ever get run without VERIFY, we will presumably need a separate define to classify "test support code".